rclrs/src/node/mod.rs

```diff
@@ -212,4 +212,38 @@ impl Node {
             .filter_map(Weak::upgrade)
             .collect()
     }
+
+    /// Returns the ROS domain ID that the node is using.
+    ///    
+    /// returned values depend on ROS_DOMAIN_ID environment variable.   
```

Comment:
I always like to include a general explanation, like this:

```suggestion
    /// The domain ID controls which nodes can send messages to each other, see the [ROS 2 concept article][1].
    /// It can be set through the `ROS_DOMAIN_ID` environment variable.
    ///
    /// [1]: https://docs.ros.org/en/rolling/Concepts/About-Domain-ID.html
```

Reply:
Thanks for your more comfortable comments!